File: lesson_1/code/task_1_1_2_fibonacci_while_loop.m
fib_seq = [1,1]; 
max_value = 5000;

%set index to 2
i = 2;

while (fib_seq(i) + fib_seq(i-1)) < max_value
    fib_seq(i+1) = fib_seq(i) + fib_seq(i-1);
    i = i + 1;
end

disp(fib_seq);

File: lesson_1/code/task_1_1_2_fibonacci_while_loop.py
fib_seq = [1, 1]
max_value = 5000
i = 1 # Python uses 0-based indexing

while (fib_seq[i] + fib_seq[i-1]) < max_value:
    fib_seq.append(fib_seq[i] + fib_seq[i-1])
    i += 1

print(fib_seq)